core/src/main/resources/com/squarespace/template/plugins/platform/quantity-input.html

```diff
@@ -1,6 +1,6 @@
 {.var @quantityLabel localizedStrings.productQuantityInputLabel}
 
 <div class="product-quantity-input" data-item-id="{id}" data-animation-role="content">
-  <div class="quantity-label">{.if @quantityLabel}{@quantityLabel}{.or}Quantity{.end}:</div>
```

Comment:
Have we considered a solution that could retain the `<div class="quantity-label">`? Since this change is going to be so disruptive (affecting potentially all 7.0 frameworks), it has a large chance of breaking custom css on old 7.0 websites or theming on dev mode sites. I understand for this change we need to change the dom structure shape at least a little, but would it be possible to keep it closer to the original shape by retaining the div maybe as a parent element of `<label>`?

Reply:
thanks for the suggestion matt - makes sense, i'm looking into it

Reply:
Per Pavel's suggestion in the ticket I updated it to use `aria-labelledby`, which should keep styling intact.